common/endpoint-shared/src/main/java/org/kaaproject/kaa/common/Constants.java

```diff
@@ -20,44 +20,38 @@
  * Common Kaa project Constants.
  */
 public interface Constants { //NOSONAR
-    /**
-     * Used URI delimiter.
-     */
-    public static final String URI_DELIM = "/"; //NOSONAR
+
+    String URI_DELIM = "/"; //NOSONAR
```

Comment:
Maybe convert Constants to the final class?
See http://stackoverflow.com/questions/2659593/what-is-the-use-of-interface-constants